python/setup.py

```diff
@@ -1,13 +1,17 @@
 # Copyright (c) 2019-2022, NVIDIA CORPORATION.
 
+import os
+
 from setuptools import find_packages
 from skbuild import setup
 
 import versioneer
 
 setup(
-    name="rmm",
-    version=versioneer.get_version(),
+    name="rmm" + os.getenv("RAPIDS_PY_WHEEL_CUDA_SUFFIX", default=""),
```

Comment:
Am wondering if there is a way to keep the `name` unchanged. Could we have a common dependency? Are there other ways to encode this information?

Reply:
Could you elaborate on the solution that you are proposing with a common dependency? The purpose of this approach is to support installation into environments with different API incompatible versions of CUDA. Specifically, once we start supporting CUDA 12 we will need wheels built specifically for the appropriate CUDA major version. Without first class support in the [wheel spec itself](https://peps.python.org/pep-0427/#file-name-convention) I am not sure how else we can handle this other than to have separate packages.